src/pages/index.tsx

```diff
@@ -1,21 +1,23 @@
+import { GetStaticProps } from 'next';
 import App from '../App';
-import { AppHead } from '../AppHead';
+import { AppHeadProps } from '../AppHead';
 
-const HomePage = () => {
-  return (
-    <>
-      <AppHead
-        title="XState Visualizer"
-        ogTitle="XState Visualizer"
-        description="Visualizer for XState state machines and statecharts"
-        importElk
-        importPrettier
-        // TODO - get an OG image for the home page
-        ogImageUrl={undefined}
-      />
-      <App />
-    </>
-  );
-};
+export const getStaticProps: GetStaticProps<{ headProps: AppHeadProps }> =
+  async () => {
+    return {
+      props: {
+        headProps: {
+          title: 'XState Visualizer',
+          ogTitle: 'XState Visualizer',
+          description: 'Visualizer for XState state machines and statecharts',
```

Comment:
Side-note: should we call this `Stately VIz` now? I can go either way.

Reply:
Sounds about right.